app/views/common_parameters/_parameter.html.erb

```diff
@@ -22,8 +22,12 @@
                         :rows => line_count(f, :value),
                         :disabled => disabled,
                         :placeholder => _("Value")) %>
+
           <% end %>
         </td>
+        <td class="col-md-1">
```

Comment:
This is a little different to the host UI, where the button's actually inside the value column, is that intended?